packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -262,10 +262,10 @@ Ultimately, Card is a very flexible component that can fit a variety of needs. I
 | Property         | Default token         | Modifiable? |
```

Comment:
Question: we've been moving away from having these anatomy sections on the docs site. instead of updating it, should we just remove the section entirely?

Reply:
Have we removed other existing ones? Sometimes I have found these helpful but I guess I usually move past them

Reply:
yeah, we've started removing them. I find them helpful, too, but they end up being hard to maintain :\

Reply:
ok, removed